### ISSUE TITLE
view menu needs indication of which views are currently active

### ISSUE THREAD

**Issue by RichCaloggero** [microsoft/vscode#121645]
@isidorn :

It seems that the view menu has toggles for various views like explorer, sidebar, etc. However, seems that there should be a checkbox state on those menus indicating which view is active.

It also isn't clear how to make these settings stick when I close vsCode.

As a screen reader user, having too many views open is distracting, confusing, and really slows me down.
If I'm working on web code, all I want on screen is one editor, and a debugging console. This is what I get if I do everything in windows : i.e. I have my text editor open and firefox, and firefox devtools.

Maybe I'm missing something...

**Comment by isidorn** [comment]
The checkbox should be read out if they are checked, however if they are unchecked there seems to be an issue and that one is covered by https://github.com/microsoft/vscode/issues/120597 fyi @sbatten 

We always try to preserve UI when you close VS Code, so they should all stick. If some of them do not then it is an error

I understand having multiple views can be distracting, I suggest just trying to close them all.
Or you can also try using Zen Mode which does just that. F1 > zen mode

Let me know how it goes and if you have other suggestions for improvement. Thanks

**Comment by RichCaloggero** [comment]
Should I close this one in lieu of issue 120597 ?

**Comment by isidorn** [comment]
Yeah I think that would be best. I can close
